packages/manager/apps/hycu/src/pages/listing/Listing.page.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+import { OsdsLink } from '@ovhcloud/ods-components/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  Datagrid,
+  DataGridTextCell,
+  useResourcesIcebergV6,
+  dataType,
+  BaseLayout,
+} from '@ovh-ux/manager-react-components';
+
+import { url } from 'inspector';
+import Loading from '@/components/Loading/Loading';
+import ErrorBanner from '@/components/Error/Error';
+import Breadcrumb from '@/components/Breadcrumb/Breadcrumb';
+
+import appConfig from '@/hycu.config';
+import { urls } from '@/routes/routes.constant';
+
+export default function Listing() {
+  const myConfig = appConfig;
+  const serviceKey = myConfig.listing?.datagrid?.serviceKey;
+  const [columns, setColumns] = useState([]);
+  const navigate = useNavigate();
+  const location = useLocation();
+  const { t } = useTranslation('listing');
+  const {
+    flattenData,
+    isError,
+    error,
+    totalCount,
+    hasNextPage,
+    fetchNextPage,
+    isLoading,
+    status,
+    sorting,
+    setSorting,
+    pageIndex,
+  } = useResourcesIcebergV6({
+    route: `/license/hycu`,
+    queryKey: ['hycu', `/license/hycu`],
+  });
+
+  const navigateToDashboard = (label: string) => {
+    const path =
+      location.pathname.indexOf('pci') > -1 ? `${location.pathname}/` : '/';
+    navigate(`${path}${label}`);
+  };
+
+  useEffect(() => {
+    if (columns && status === 'success' && flattenData?.length > 0) {
+      const newColumns = Object.keys(flattenData[0])
+        .filter((element) => element !== 'iam')
+        .map((element) => ({
+          id: element,
+          label: element,
+          type: dataType(flattenData[0][element]),
+          cell: (props: any) => {
+            const label = props[element] as string;
+            if (typeof label === 'string' || typeof label === 'number') {
```

Comment:
As is `any` type with external element. We need protect call (?)

```suggestion
  useEffect(() => {
    if (columns && status === 'success' && flattenData?.length > 0) {
      const newColumns = Object.keys(flattenData[0])
        .filter((element) => element !== 'iam')
        .map((element) => ({
          id: element,
          label: element,
          type: dataType(flattenData[0][element]),
          cell: (props: any) => {
            const label = props?.[element] as unknown;
            if (typeof label === 'string' || typeof label === 'number') {
```

Reply:
will be removed later